### ISSUE TITLE
New builds not appearing on update list

### ISSUE THREAD

**Issue by zur003** [APSIMInitiative/ApsimX#3268]
The "Upgrade" dialog is not show any versions after the end of October.

**Comment by hol430** [comment]
Yeah the upgrade system is offline at the moment. Am working on it at the moment, but I can generate a manual release if you want a new update.

**Comment by zur003** [comment]
I don't need a new release myself - it's easy enough to re-compile. Just wanted to be sure you and others were aware that upgrades were broken.
